### PR TITLE
[Migrate to SQLAlchemy ORM - PART 2] Create SQLAlchemy models with helper methods and initiate connection through ORM

### PR DESCRIPTION
**Pull request by thomasmunduchira**

### CODE REVIEW

File: src/shared.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+class Shared:
+    db = None
+    google = None
+    login_manager = None
+
+    @classmethod
+    def instantiate(cls, app):
+        """Instantiate shared data"""
+        cls.db = SQLAlchemy()
+        importlib.import_module('src.models.Answer')
```
Comment (klanmiko) [inline src/shared.py]: why is this here?

File: app.py [hunk 2]
```diff
@@ -1,20 +1,24 @@
...
 app = Flask(__name__)
 
 # load config
 app.config.from_object('src.config.config.Config')
 app.secret_key = app.config.get('SECRET_KEY')
+
+# instantiate shared data
+Shared.instantiate(app)
+
+from src.controller.admin import admin
```
Comment (klanmiko) [inline app.py]: why are these here?

File: src/models/Question.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    def get_questions_from_csv(csv_file):
+        """Get questions from CSV file"""
+        reader = csv.DictReader(csv_file)
+        return reader.fieldnames
+
+    @staticmethod
+    def convert_questions_to_rows(questions):
+        """Convert questions into rows to insert into database"""
+        return [
+            Question.convert_question_to_row(question_tuple[0] + 1, question_tuple[1])
```
Comment (klanmiko) [inline src/models/Question.py]: what are question_tuple[0], question_tuple[1]. Mark it in code

File: src/models/enums/QuestionType.py [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import enum
+
+
+class QuestionType(enum.Enum):
+    ignore = 'ignore'
+    essay = 'essay'
+    multiple_choice = 'multipleChoice'
```
Comment (klanmiko) [inline src/models/enums/QuestionType.py]: that's ugly, can you auto it pls and just use the name

File: src/models/enums/Role.py [hunk 5]
```diff
@@ -0,0 +1,6 @@
+import enum
+
+
+class Role(enum.Enum):
+    user = 'user'
+    admin = 'admin'
```
Comment (klanmiko) [inline src/models/enums/Role.py]: same here

File: app.py (hunk 2 above)
Comment (thomasmunduchira) [reply app.py]: The controllers depend on the shared data instances being instantiated, so I'm importing them after.

File: src/models/Question.py (hunk 3 above)
Comment (thomasmunduchira) [reply src/models/Question.py]: Done

File: src/models/enums/QuestionType.py (hunk 4 above)
Comment (thomasmunduchira) [reply src/models/enums/QuestionType.py]: Done

File: src/models/enums/Role.py (hunk 5 above)
Comment (thomasmunduchira) [reply src/models/enums/Role.py]: Done

File: src/shared.py (hunk 1 above)
Comment (klanmiko) [reply src/shared.py]: ok

File: app.py (hunk 2 above)
Comment (klanmiko) [reply app.py]: seems like a circular reference but okie

**Comment by klanmiko** [comment]
@thomasmunduchira fix the merge conflict please
